package.json

```diff
@@ -60,6 +60,7 @@
     "moment": "^2.24.0",
     "no-scroll": "^2.1.1",
     "register-service-worker": "^1.6.2",
+    "serialize-javascript": ">=2.1.1",
```

Comment:
what is this for / is it needed?

Reply:
good catch. don't know where this comes from and will remove it

Reply:
this is also in the resolution (to prevent security vulnerable deps) will leave it in and create another PR